docs/index.adoc

```diff
@@ -150,53 +150,38 @@ It *must be unique* for your application.
 
 NOTE: See <<Feature Permissions>> for details on the various states a feature can have.
 
-=== Strategy Splits and general targeting rules
+== Rollout strategies and targeting rules
 
-Each feature in an environment always has a default value. You can also apply zero or more
-rollout strategies to this feature. e.g. a feature flag in each environment has one default value 
-and can have zero or more rollout strategies.
+=== Rollout strategies
 
-A rollout strategy consists of one or more rules. That rule can consist of any combination of matching criteria, so we refer to them
-often as targeting rules. e.g. if you want to turn your feature on for a user only on a mobile device or 
-only with a specific email or userIds. 
-
-==== Each rule is an AND condition
-
-Each additional rule is always applied as an AND condition - the user is using a mobile device that is iOS *AND*
-their country is Australia.
+Rollout strategies provide an ability to rollout features to a limited audience based on targeting rules, for example imagine you have a feature flag of type string which controls a "submit button color" that can be in multiple states,
+e.g green, blue, red etc. With rollout strategies, you can serve a `green` value to users on `iOS` devices,
+`blue` value to users whose emails ending with `gmail.com` and `red` value to users who are in
+`New Zealand or United States or United Kingdom`. You can also use percentage based rollouts and for example, turn your feature "on" only to 50% of the audience.
 
-==== Percentage Splits rules
+When you create a feature and set initial value, it is treated as a default strategy and a default value (set per each environment). If you decide to split your rollout you can add a rollout strategy and set a feature value for this strategy different from the default.
```

Comment:
this paragraph is pretty wishy-washy. It isn't clear what is a feature value, what is a rollout strategy, what is a default. It seems to introduce a whole lot of terms in one sentence without defining them.